akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -544,15 +548,15 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     val (ctrl, completed) =
       if (remoteSettings.TestMode) {
```

Comment:
All I could say is to measure... Nevertheless, it should not add meaningful bottleneck at the transmission rate we are currently at, but small things can add up.